model/index.js

```diff
@@ -61,28 +62,37 @@ module.exports = yeoman.Base.extend({
   addNullDataSourceItem: actions.addNullDataSourceItem,
 
   setBluemixDatasourceState: function() {
+    if (this.abort) return;
     if (this.options.bluemix) {
       var configPath = this.destinationPath('.bluemix/datasources-config.json');
       if (!fs.existsSync(configPath)) {
-        this.log('datasources-config.json not found');
-        process.exit();
-      }
-      var datasourcesConfig = JSON.parse(fs.readFileSync(configPath));
-      this.bluemixDataSourcesList = datasourcesConfig.datasources;
-      if (Object.keys(this.bluemixDataSourcesList).length) {
-        this.hasDatasources = true;
+        this.abort = true;
```

Comment:
Raymond and I looked into the possibility last time, couldn't find one. Do you have any suggestion?